python_modules/dagster/dagster/_core/storage/event_log/in_memory.py

```diff
@@ -352,6 +373,29 @@ def get_asset_run_ids(self, asset_key):
 
         return list(asset_run_ids)
 
+    def get_asset_event_tags(self, asset_key: AssetKey) -> Sequence[Tuple[str, AbstractSet[str]]]:
+        tags: Dict[str, Set] = defaultdict(set)
+        for records in self._logs.values():
+            for record in records:
+                if (
+                    record.is_dagster_event
+                    and record.dagster_event.asset_key == asset_key
+                    and record.dagster_event.event_type_value
+                    == DagsterEventType.ASSET_MATERIALIZATION.value
+                    and self._wiped_asset_keys.get(record.dagster_event.asset_key, 0)
+                    < record.timestamp
+                    and record.dagster_event.step_materialization_data.materialization.tags
+                ):
+                    for (
+                        key,
+                        value,
```

Comment:
I think this could be laid out on fewer lines

Reply:
black formatted this 🤷‍♀️

Reply:
if you take out the comma after value, it might compress them

Reply:
still does not compress, not sure why

Reply:
weird. ok no worries